xbmc/pvr/PVRManager.cpp

```diff
@@ -627,15 +627,16 @@ bool CPVRManager::Load(void)
   /* start the add-on update thread */
   if (m_addons)
     m_addons->Start();
+  else
+    return false;
 
-  /* load at least one client */
-  while (IsInitialising() && m_addons && !m_addons->HasConnectedClients())
-    Sleep(50);
```

Comment:
Since the focus of this PR is to get something in that works, can we agree on a suitable timeout and focus on refactoring this properly some time in the future?